src/database/build.js

```diff
@@ -0,0 +1,16 @@
+const fs = require('fs');
+
+const buildDatabase = () => {
+  const connection = require('./dbConnection');
```

Comment:
calling anther file 'dbConnection' it must out of your function

Reply:
done